@navikt/core/css/darkside/form/switch.darkside.css

```diff
@@ -181,12 +158,13 @@
   cursor: not-allowed;
 }
 
-/* Readonly */
+/* -------------------------- Switch Readonly state ------------------------- */
 .navds-switch--readonly > .navds-switch__track,
```

Comment:
Consider adding all this nested inside .navds-switch--readonly to avoid repitition